cpp/src/arrow/compute/api_scalar.h

```diff
@@ -1332,6 +1337,57 @@ Result<Datum> ISOYear(const Datum& values, ExecContext* ctx = NULLPTR);
 ARROW_EXPORT
 Result<Datum> USYear(const Datum& values, ExecContext* ctx = NULLPTR);
 
+/// \brief PGYear returns PostgreSQL year for each element of `values`.
+/// First week of PostgreSQL epidemiological year has the majority (4 or more) of it's
+/// days in January. Last week of PostgreSQL epidemiological year has the year's last
+/// Wednesday in it. PostgreSQL epidemiological week starts on Sunday.
```

Comment:
It seems that this is the same description as `USYear()`. Is there any difference between `USYear()` and `PGYear()`?